src/Client/ConfigWindow.java

```diff
@@ -802,6 +801,28 @@ public void actionPerformed(ActionEvent e) {
     generalPanelCameraMovableRelativeCheckbox.setToolTipText(
         "Camera movement will follow the player position");
 
+    addSettingsHeader(generalPanel, "Menu/Item patching");
+
+    generalPanelBypassAttackCheckbox = addCheckbox("Always left click to attack", generalPanel);
+    generalPanelBypassAttackCheckbox.setToolTipText(
+        "Left click attack monsters regardless of level difference");
+    generalPanelBypassAttackCheckbox.setBorder(new EmptyBorder(7, 0, 10, 0));
+
+    generalPanelCommandPatchEdibleRaresCheckbox =
+        addCheckbox("Disable the ability to ingest holiday items or rares", generalPanel);
+    generalPanelCommandPatchEdibleRaresCheckbox.setToolTipText(
+        "Applies to the Easter Egg, Pumpkin, and the Half Wine.");
+
+    generalPanelCommandPatchQuestCheckbox =
+        addCheckbox("Swap eat & use options on Quest Items", generalPanel);
+    generalPanelCommandPatchQuestCheckbox.setToolTipText(
+        "Applies to giant Carp, chocolaty milk, Rock cake, and nightshade.");
+
+    generalPanelCommandPatchDiskOfReturningCheckbox =
+        addCheckbox("Remove the spin option from the Disk of Returning", generalPanel);
+    generalPanelCommandPatchDiskOfReturningCheckbox.setToolTipText(
+        "There is no reason to want to do this. Kept in RSC+ as a historic option.");
```

Comment:
That comment is valid or could be reworded that simulates RS2's disk since they didn't have spin option (https://youtu.be/yKCm5kfSfqo?t=50), could say not recommended if the player is in a world with active black hole

Reply:
I wouldn't say about RS2 personally, seems not relevant to RSC.
Would also say that if there is a world with an active black hole, the disk of returning wouldn't be discontinued, so it's OK to consume it.